language/move-vm/runtime/src/loader.rs

```diff
@@ -2032,6 +2188,7 @@ impl Function {
             type_parameters,
             native,
             def_is_native,
+            def_is_friend_or_private,
```

Comment:
I haven't looked where this is used, but wouldn't you want to know the exact visibility?